### PR TITLE
KBD67 MKII Notes Update

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

This PR has no logic change. Though my sentences may have questionable logic. 

We've had quite a few people not sure which firmware to use for what board. Hopefully these notes will help guide people to flashing .bin files and .hex files on the appropriate boards. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd67/mkiirgb/readme.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
-Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)  
-Hardware Supported: KBD67 Mk.II RGB  
-Hardware Availability: [kbdfans](https://kbdfans.myshopify.com/)
+* Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)
+* Hardware Supported: KBD67 Mk.II RGB V1 and V2
+* Hardware Availability: [kbdfans](https://kbdfans.myshopify.com/)
 
+There are two versions of the KBD67 MKII RGB. Please use the appropriate firmware for your board. 
+
+V1: ARM
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/mkiirgb/readme.md]:
```suggestion
V1: STM32F303 (ARM)
```

File: keyboards/kbdfans/kbd67/mkiirgb/readme.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
+V2: Atmega32u4
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/mkiirgb/readme.md]:
```suggestion
V2: ATmega32U4 (AVR)
```

File: keyboards/kbdfans/kbd67/mkiirgb/readme.md [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
 
 Make example for this keyboard (after setting up your build environment):
 
-    make kbdfans/kbd67/mkiirgb:default
+    make kbdfans/kbd67/mkiirgb/v1:default
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/mkiirgb/readme.md]:
```suggestion
    make kbdfans/kbd67/mkiirgb/v1:default # Arm (STM32F303)
```

File: keyboards/kbdfans/kbd67/mkiirgb/readme.md [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
+    make kbdfans/kbd67/mkiirgb/v2:default
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/mkiirgb/readme.md]:
```suggestion
    make kbdfans/kbd67/mkiirgb/v2:default # AVR (ATmega32U4)
```

File: keyboards/kbdfans/kbd67/readme.md [hunk 5]
```diff
@@ -1,22 +1,28 @@
...
-3. Rev2: Released in April/May 2019, the Rev2 also needs to be soldered together and supports multiple layouts. 
-4. MKII RGB: Released in September 2019. ARM powered hotswap board. 
+A 65% keyboard sold in several variants. 
+
+1. Rev1: Released in early 2018. Atmega32u4 soldered keyboard. Sold under the name "KBD65".  
+2. HotSwap: Released in late 2018. Atmega32U4 hotswap single layout keyboard. 
+3. Rev2: Released in April/May 2019. Atmega32u4 soldered keyboard.
+4. MKII: Released in September 2019, designed by ai03. Atmega32u4 soldered keyboard. ESD Protected
+5. MKII RGB V1: Released in September 2019. ARM hotswap board. 
+6. MKII RGB V2: Released in December 2019. Atmega32u4 hotswap board. ESD Protected.
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/readme.md]: IMO this section should link to the appropriate revision folder, similar to how the Clueboard vendor readme works.

https://github.com/qmk/qmk_firmware/blob/8fa64568f01604bde0bb5eb1923e25483af03424/keyboards/clueboard/readme.md

Comment (mechmerlin) [reply keyboards/kbdfans/kbd67/readme.md]: Not a bad idea

**Pull request merged**